src/config.h

```diff
@@ -44,5 +47,15 @@ class Config {
   CoreConfig core;
   AuthConfig auth;
   DeviceConfig device;
+
+ private:
+  boost::property_tree::ptree pt;
+
+  std::string getStringValue(const std::string &name) {
```

Comment:
I don't  think so, this is private member function and should be accessible only in this class